cmd/backend/main.go

```diff
@@ -60,16 +60,18 @@ func main() {
 	log.SetOutput(f)
 	log.SetFormatter(&log.JSONFormatter{})
 
-	ticksPerDay := 24 * 60
-	simDays := 3
-	reshuffleDays := 1
-	dayInfo := day.NewDayInfo(ticksPerDay, simDays, reshuffleDays)
-
 	// can have frontend parameters come go straight into simEnv
 	foodOnPlatform := 100.0
 	numOfAgents := []int{2, 2, 2}
 	agentHP := 100
 	agentsPerFloor := 1 //more than one not currently supported
+	numberOfFloors := simulation.Sum(numOfAgents) / agentsPerFloor
```

Comment:
I wasn't at the meeting so I can't comment on the decision of which variables are inputs, but what's the guarantee `numberOfFloors` is an integer here?